Defs/Tutor/Concepts_NotedOpportunistic.xml

```diff
@@ -122,7 +122,7 @@

```

Comment:
Learning helper content should be concise, for the following reasons:
1. There's no scrolling support, so any part of the learning helper that goes off-screen is inacessible. You've put so much text in here that it's missing four entire paragraphs when the game is running at 1920x1080 with 1.0x UI scale.
2. A 3,000-character block of text is hard to take for a new player just trying to learn the CE basics. If you take the previous text and just edit it slightly to mention that a projectile progressively converts part of its sharp damage to blunt as a target's armor gets closer to the projectile's armor penetration, that should be informative enough to get the ball rolling. If later on they want to dive into the specifics, they can check the armor info panels, which is where you should provide an in-depth explanation of sharp/blunt armor penetration and deflection. You can edit them at `CombatExtended\Patches\Core\Stats\Stats.xml`

Reply:
Changed the learning helper text, now it's only twice as big (as opposed to four times), at the same time edited the description of the sharp armor stat.